### PR TITLE
Add new post with full title for non-voting member

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR handles part of https://github.com/datamade/la-metro-councilmatic/issues/327

### CODE REVIEW

File: lametro/__init__.py [hunk 1]
```diff
@@ -44,6 +44,10 @@ def get_organizations(self):
                      'Nonvoting Board Member',
                      division_id='ocd-division/country:us/state:ca')
 
+        org.add_post('Acting Caltrans District 7 Director, Appointee of Governor of California',
```
Comment (fgregg) [inline lametro/__init__.py]: Is this really the post? Particularly the "acting" bit?

**Comment by fgregg** [comment]
To me it is not clear that

Acting Caltrans District 7 Director, Appointee of Governor of California

is a post.

Will there always be somebody with that title? If there is someone without
that title, does that mean there's vacancy for this position?

Best,

Forest

**Comment by fgregg** [comment]
Let me know if you want to talk through the difference between a post and a member. It's something I have gotten hung up on in the past.

**Comment by reginafcompton** [comment]
@fgregg - this would be good to talk through.

**Pull request merged**